dubbo-spring-boot/dubbo-spring-boot-autoconfigure/src/main/java/org/apache/dubbo/spring/boot/autoconfigure/DubboTripleAutoConfiguration.java

```diff
@@ -56,4 +59,24 @@ public FilterRegistrationBean<TripleFilter> tripleProtocolFilter(
             return registrationBean;
         }
     }
+
+    @Configuration(proxyBeanMethods = false)
+    @ConditionalOnClass(Filter.class)
+    @ConditionalOnWebApplication(type = Type.SERVLET)
+    @ConditionalOnProperty(prefix = WEBSOCKET_PREFIX, name = "enable")
```

Comment:
Use enabled to keep the same wording style as spring

Reply:
Need add into https://github.com/apache/dubbo/blob/3.3/dubbo-common/src/main/java/org/apache/dubbo/config/nested/TripleConfig.java#L146 like servlet for generate spring config autocomplete metadata